firstvoices/backend/views/category_views.py

```diff
@@ -58,6 +60,20 @@
                     ),
                 ],
             ),
+            OpenApiParameter(
+                name="nested",
+                description=_(
+                    "Returns a nested list of categories with their children if enabled"
+                ),
+                # If this parameter is true, ParentCategoryFlatListSerializer is used for the response
```

Comment:
Notes related to this added in ticket [FW-4660](https://firstvoices.atlassian.net/browse/FW-4660)